### PR TITLE
dashboard: link to picasso

### LINKED ISSUE #5064: Dashboard: Add links on the periods to the picasso of this period

### PR DESCRIPTION
**Pull request by pmattmann**
fixes #5064

### CODE REVIEW

File: frontend/src/views/camp/Dashboard.vue [hunk 1]
```diff
@@ -25,10 +25,13 @@
           class="mx-4 mt-6 mb-3"
           style="border-collapse: collapse"
         >
-          <caption class="font-weight-bold text-left">
-            {{
-              periods[uri].description
-            }}
+          <caption class="text-left">
+            <router-link
```
Comment (carlobeltrame) [inline frontend/src/views/camp/Dashboard.vue:29]: I can't seem to focus this link with the keyboard (by tabbing), why is that?

**Review by carlobeltrame — APPROVED** [review]

File: frontend/src/views/camp/Dashboard.vue (hunk 1 above)
Comment (manuelmeister) [reply frontend/src/views/camp/Dashboard.vue:29]: Can't reproduce your problem

**Review by manuelmeister — APPROVED** [review]
Interesting idea

File: frontend/src/views/camp/Dashboard.vue (hunk 1 above)
Comment (carlobeltrame) [reply frontend/src/views/camp/Dashboard.vue:29]: Hm, it only happens on Firefox for me... Let's leave it for now

**Pull request merged**